api/src/services/server.ts

```diff
@@ -94,6 +94,30 @@ export class ServerService {
 			};
 		}
 
+		if (this.accountability?.user) {
+			if (env['WEBSOCKETS_ENABLED']) {
```

Comment:
thoughts on moving the whole `if (env['WEBSOCKETS_ENABLED'])` into the same `if (this.accountability?.user)` in line 54 above?

Reply:
the outer `if` is there to guard against the websocket key being set when requesting server info unauthenticated. if we were to merge those it would always be set.

Reply:
I was wondering about moving it into the existing authenticated block here: https://github.com/directus/directus/blob/80fb6010b032888c2b3183540eff4b0253098fd7/api/src/services/server.ts#L54-L75, so we'll have the same "authenticated-only keys" in the same IF block 👍 sorry for the confusion 😄

Reply:
We definitely could do that, i dont have strong oppinions either way for this change 😬

Reply:
Likewise! But let's table this for now since it isn't really something crucial 😄